src/renderer/services/chain/fees/swap.ts

```diff
@@ -57,23 +71,66 @@ const feesByChain$ = ({
         liveData.map((btcFees) => btcFees.fees)
       )
 
-    case ETHChain:
+    case ETHChain: {
+      // deposit fee estimation requires recipient/router address and amount
+      let isValid = false
+
+      if (!recipient || !validateAddress(recipient)) {
+        isValid = true
+      }
+      if (!amount || amount.amount().eq(0)) {
+        isValid = true
+      }
```

Comment:
Why all this validation is needed here? It will be done by using `sequenceSOption` later in code.